auth0-cis-webhook-consumer.yaml

```diff
@@ -212,7 +257,7 @@ Resources:
         Variables:
           FUNCTION_NAME: !Ref Auth0CISWebHookConsumerAsyncFunction
       Handler: index.lambda_handler
-      Runtime: python3.7  # AWS Lambda doesn't support inline code for python3.8
```

Comment:
This deletes the comment for "here's why we're using an old py version."  Since py3.7 is EOL in Lambdas, are we in future-trouble here?  (real question: should this deletion-of-comment cause another issue to be opened to investigate something?)

Reply:
Good catch....I have been updating sso-dashboard to 3.12. I will take a look.

Reply:
updated conversation